scripts/cat_relations/relationship.py

```diff
@@ -99,9 +99,9 @@ def start_interaction(self) -> None:
         all_interactions = NEUTRAL_INTERACTIONS.copy()
         if in_de_crease != "neutral":
             all_interactions = INTERACTION_MASTER_DICT[rel_type][in_de_crease].copy()
-            possible_interactions = self.get_relevant_interactions(all_interactions, intensity, biome, season, game_mode)
+            possible_interactions = self.get_relevant_interactions(all_interactions, intensity, biome, season, game_mode, False)
```

Comment:
Rather than creating a new parameter into this function, I think it would be better to pass `None` for intensity in the case of neutral interactions. In the function itself, simply ignore the intensity check if intensity is `None`.